jira/jira.go

```diff
@@ -123,15 +142,17 @@ func promptForAuth(ctx *ankh.ExecutionContext, retryCount int) (string, string,
 	return providedUsername, providedPassword, nil
 }
 
-func getSummary(ctx *ankh.ExecutionContext, env string) (string, error) {
+func getSummary(ctx *ankh.ExecutionContext, chart *ankh.Chart, envOrContext string) (string, error) {
 	// If format is set, use that
 	format := ctx.AnkhConfig.Jira.SummaryFormat
 	if ctx.Mode == ankh.Rollback {
 		format = ctx.AnkhConfig.Jira.RollbackSummaryFormat
 	}
 
+	chartName := fmt.Sprintf("%v@%v", chart.Name, chart.Version)
```

Comment:
one tricky bit: `chart.Version` could be empty, and when it is, `chart.Path` is non-emtpy. it refers to the local filesystem path that was used to apply the chart. we could simply log that, though it's probably not meaningful to the slack channel. at the very least it lets them know the user did a local, non-versioned operation.